src/main/kotlin/lotto/data/WinningLotto.kt

```diff
@@ -19,6 +19,14 @@ data class WinningLotto(val lotto: Lotto, val bonusNumber: LottoNumber) {
     }
 
     private fun validateDuplicationBonusNumber() {
-        require(!lotto.selectNumbers.contains(bonusNumber)) { "당첨 번호 구성과 보너스 번호가 중복됩니다." }
+        require(!lotto.selectNumbers.contains(bonusNumber)) { ERR_MSG_DUPLICATION_BONUS_NUMBER }
```

Comment:
좀 더 객체 지향적으로 설계가 되었다면 다음과 같이 작성될 수 있을꺼에요. 
```suggestion
        require(!lotto.containNumber(bonusNumber)) { ERR_MSG_DUPLICATION_BONUS_NUMBER }
```